platform/darwin/src/MGLSource.h

```diff
@@ -16,10 +16,10 @@ NS_ASSUME_NONNULL_BEGIN
  add and remove sources dynamically using methods such as
  `-[MGLStyle addSource:]` and `-[MGLStyle sourceWithIdentifier:]`.
 
- Do not create instances of this class directly, and do not create your own
- subclasses of this class. Instead, create instances of `MGLShapeSource`, 
- `MGLImageSource` and the concrete subclasses of `MGLTileSource`,
```

Comment:
Will keep the release branch comment and add `MGLImageSource` after `MGLShapeSource`